scala/StalinSortSimple.scala

```diff
@@ -0,0 +1,20 @@
+import scala.util.Random
+
+object StalinSort {
```

Comment:
we already have a scala solution that uses filter, do you want to suggest an improvement on that or merge only the recursive? (the recursive we don't have yet)

Reply:
oh...
drop this Pull request.
because, https://github.com/gustavo-depaula/stalin-sort/pull/54/files is better than this recursive-code.
and https://github.com/gustavo-depaula/stalin-sort/blob/master/scala/StalinSortWithFilter.scala is compatible many types.

Reply:
ok! thanks for your effort anyway